src/status_im/ui/screens/about_app/styles.cljs

```diff
@@ -17,3 +17,13 @@
 
 (def about-title-text
   {:font-size 20})
+
+(def learn-more-title
+  {:font-size   17
+   :line-height 20
+   :font-weight "600"})
+
+(def learn-more-text
+  {:color       colors/gray
+   :font-size   15
```

Comment:
this is default typography no need to specify font size and line height

Reply:
Done.